templates/firefox-ios.xliff

```diff
@@ -1778,18 +1818,54 @@
         <source>Bookmark Removed</source>
         <note>Toast displayed to the user after a bookmark has been removed.</note>
       </trans-unit>
+      <trans-unit id="Menu.RemoveBookmark.Label.v99" xml:space="preserve">
+        <source>Remove</source>
+        <note>Label for the remove bookmark button in the menu. Pressing this button remove the current page from the bookmarks. Please keep the text as short as possible for this label.</note>
+      </trans-unit>
       <trans-unit id="Menu.RemovePin.Confirm" xml:space="preserve">
         <source>Removed From Top Sites</source>
         <note>Toast displayed to the user after removing the item from the Top Sites.</note>
       </trans-unit>
-      <trans-unit id="Menu.RemovePin.Confirm2" xml:space="preserve">
+      <trans-unit id="Menu.RemovePin.Confirm2.v99" xml:space="preserve">
         <source>Removed from Shortcuts</source>
         <note>Toast displayed to the user after removing the item to the Shortcuts.</note>
       </trans-unit>
+      <trans-unit id="Menu.RemoveReadingList.AlternateLabel.v99" xml:space="preserve">
+        <source>Menu.RemoveReadingList.AlternateLabel.v99</source>
```

Comment:
Seems like this is the string ID instead of the actual string ;)

Reply:
Oups let me look at this! Thanks!!

Reply:
It should be ok now 🤞